heron/metricsmgr/src/java/com/twitter/heron/metricsmgr/sink/FileSink.java

```diff
@@ -64,25 +64,20 @@
 
   @Override
   public void init(Map<String, Object> conf, SinkContext context) {
-    filenamePrefix = (String) conf.get(FILENAME_KEY) + "." + context.getMetricsMgrId();
+    filenamePrefix = conf.get(FILENAME_KEY) + "." + context.getMetricsMgrId();
```

Comment:
Can we check and throw exceptions here if config does not contain "filename-output" or "file-maximum"? Or provide some defaults values?